packages/docs-theme/src/components/documentation.tsx

```diff
@@ -278,6 +278,8 @@ function getScrolledReference(offset: number, container: HTMLElement, scrollPare
  * Scroll the scrollParent such that the reference heading appears at the top of the viewport.
  */
 function scrollToReference(reference: string, container: HTMLElement, scrollParent = document.scrollingElement) {
+    // without rAF, on initial load this would scroll to the bottom because the CSS had not been applied.
```

Comment:
I think this came up in another PR a couple months ago -- still highly suspicious! why is CSS not on the page?